assets/img/icon-mobilepay-simple.svg

```diff
@@ -0,0 +1,38 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I want outlined Material icons to be supported by [jekyll-material-icon-tag](https://github.com/SwedbankPay/jekyll-material-icon-tag/) instead of adding them here.

Reply:
@NullableInt

Reply:
Choice of material icon is setting a font style on the element, should that be handled by [jekyll-material-icon-tag](https://github.com/SwedbankPay/jekyll-material-icon-tag/) or locally? The ruby code does not have any concept of font choice at the moment.

Reply:
Shouldn't this be changed in the Design Guide, then? @pthoang what do you say?

Reply:
I might have misunderstood, we can change it manually as well

Reply:
Try `class='material-icons-outlined` and see if it works.

Reply:
`class="material-icons-outlined"` should work if css for outlined material icons is imported. Whether Design Guide will include this is not decided just yet. Meanwhile, use `<link href="https://fonts.googleapis.com/css?family=Material+Icons+Outlined" rel="stylesheet">` to try it out (see https://stackoverflow.com/questions/50303454/how-to-use-the-new-material-design-icon-themes-outlined-rounded-two-tone-and for more information)

Reply:
Ok, then we need to include the above `<link>` in the theme, @NullableInt.